pangeo_forge_recipes/transforms.py

```diff
@@ -400,4 +421,7 @@ def expand(self, datasets: beam.PCollection) -> beam.PCollection:
             target=self.get_full_target(), target_chunks=self.target_chunks
         )
         rechunked_datasets | StoreDatasetFragments(target_store=target_store)
+        if self.consolidate_coords:
+            ConsolidateDimensionCoordinates(target_store=target_store)
```

Comment:
```suggestion
            target_store | ConsolidateDimensionCoordinates()
```
@norlandrhagen hmm maybe as a PTransform, ConsolidateDimensionCoordinates needs to take a PCollection as input (via `|`)? If so, we can probably do it like this.